### PR TITLE
Adjustments lp

### PR DESCRIPTION
**Pull request by JbellonDev**
Notification file fixed.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "livePreview.defaultPreviewPath": "/notes.html"
```
Comment (hrodriguezimagina) [inline .vscode/settings.json:2]: remove this change it's not related to the project

File: css/layout.css [hunk 2]
```diff
@@ -406,7 +415,204 @@ body {
...
+  }
+
+.custom-nav-tabs .nav-link {
+  height: 50px;
+  line-height: 50px;
+  display: flex;
+  align-items: center;
+}
+
+.vl {
```
Comment (hrodriguezimagina) [inline css/layout.css]: what this name means?

File: js/jquery-ui.js
Comment (hrodriguezimagina) [inline js/jquery-ui.js:1]: undone this , why do you modify an external dependency?

**Review by hrodriguezimagina — CHANGES_REQUESTED** [review]
plase solve the comments left on this pr

File: css/layout.css (hunk 2 above)
Comment (hrodriguezimagina) [reply css/layout.css]: vl and where did you use it ?

File: js/jquery-ui.js
Comment (JbellonDev) [reply js/jquery-ui.js:1]: Can you confirm me please, what was it the change?

File: .vscode/settings.json (hunk 1 above)
Comment (JbellonDev) [reply .vscode/settings.json:2]: Removed

File: css/layout.css (hunk 2 above)
Comment (JbellonDev) [reply css/layout.css]: vl Removed, not tagged in any file.

**Review by hrodriguezimagina — APPROVED** [review]
aprovado por chayanne

**Pull request merged**